### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, I:
📃 Created a method called current_user in the ApplicationController to make current user data available to all controllers.
✔️ It returns the first user from the database. (without worrying about proper authentication at this time)
📃 Created forms to perform the following functions:
✔️ Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
✔️ Creates a comment on behalf of the current_user (use the method that you created in your ApplicationController).
📃 Allowed Users to add likes to Posts.

### CODE REVIEW

File: app/views/posts/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <div class="form-group">
+    <%= form.label :title %>
+    <%= form.text_field :title, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= form.label :text %>
+    <%= form.text_area :text, class: "form-control" %>
+  </div>
+  <%= form.submit 'Create Post', class: "btn btn-primary" %>
+<% end %>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/_form.html.erb]: - [ ] On submit of form I had an error, and this also hindered me from testing comments and likes, kindly fix this
Screenshot from 2023-11-03 18-30-54

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @diarisdiakite  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/_form.html.erb (hunk 1 above)
Comment (diarisdiakite) [reply app/views/posts/_form.html.erb]: Thank you @VICTOR-CHUKWU for the comment. The code has been updated.

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @YoutLone     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  . 
Congratulations! 🎉  Keep up the Awesome work 🥂  Keep Rocking 🥇 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] Professional README file  
- [x] Forms added properly






 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**